### ISSUE TITLE
questiona about instance_class_ids

### ISSUE THREAD

**Issue by J6mesQian** [MCG-NJU/SparseOcc#6]
Hi, I found out that the instance_class_ids during the training are all empty lists, which caused the program to abort. I was wondering if you have encountered this problem before. Thanks!

(Pdb) instance_class_ids
[tensor([], device='cuda:0', dtype=torch.int64), tensor([], device='cuda:0', dtype=torch.int64), tensor([], device='cuda:0', dtype=torch.int64), tensor([], device='cuda:0', dtype=torch.int64), tensor([], device='cuda:0', dtype=torch.int64), tensor([], device='cuda:0', dtype=torch.int64), tensor([], device='cuda:0', dtype=torch.int64), tensor([], device='cuda:0', dtype=torch.int64)]

**Comment by afterthat97** [comment]
`instance_class_ids` belongs to the panoptic version. I released this part a few minutes ago, please check it!

**Comment by YANG-CY-163** [comment]
This is caused by the incorrect behavior of `torch.bincount` when computing loss(may occur in PyTorch 1.10), refer to [this](https://discuss.pytorch.org/t/torch-bincount-behaves-differently-on-cpu-and-gpu/150862) . Using a higher version of PyTorch(e.g. 2.0.0) can solve this problem, hope it helps you.

**Comment by J6mesQian** [comment]
update torch does solve this, thx!!
